### DISCUSSION TITLE
Help me please. Wazuh API access on Browser

### DISCUSSION THREAD

**Discussion by admarnelson** [Security-Onion-Solutions/securityonion#7838] [Unsupported Versions] [answered]
Mates, I would like to access wazuh from browsers. the current Security onion version is 2.3.110 I followed documentation this screenshot about wazuh API. without any mistake.

![image](https://user-images.githubusercontent.com/93060289/164532435-bdd9dd4a-a470-4683-865a-a84d499b66bd.png)

My question is: Why doesn't work? any step I miss? I  put on browser https://IP_SecOnion:55000 result only something like this

{"error":0,"data":{"msg":"Welcome to Wazuh HIDS API","api_version":"v3.13.1","hostname":"securityonion-is-the-coolest","timestamp":"Wed Feb 02 2022 13:09:03    GMT+0000 (UTC)"}}

Please give me a insight

Best Regard

**Comment by admarnelson** [comment]
I also permit traffic on firewall

**Comment by dougburks** [comment]
> I put on browser https://IP_SecOnion:55000 result only something like this
> 
> {"error":0,"data":{"msg":"Welcome to Wazuh HIDS API","api_version":"v3.13.1","hostname":"securityonion-is-the-coolest","timestamp":"Wed Feb 02 2022 13:09:03 GMT+0000 (UTC)"}}

That output appears to match what the documentation screenshot says you should receive.

**Reply by admarnelson** [reply]
yeah but on output show me something diferent of course. it is just example but if need I can provide it. I just need know how to use wazuh from console web browser. please

**Reply by admarnelson** [reply]
the following screenshot is about the real my Security Onion

**Comment by admarnelson** [comment]
this is my SecOnion Screenshot

![image](https://user-images.githubusercontent.com/93060289/164885740-92c25a60-345d-4beb-9506-95567739f8d5.png)

about Deployment

![image](https://user-images.githubusercontent.com/93060289/164885858-9aa7a3f6-dc39-4832-aa8d-498fb7ca43b8.png)

again I need to acces web interface wazuh for this IP https://172.16.2.115:55000
What I need to Next Step. please help me
please
best regards

**Comment by admarnelson** [comment]
![image](https://user-images.githubusercontent.com/93060289/164886885-7fc3d02e-d944-493a-8a6c-951346f217f2.png)

so-status

**Comment by admarnelson** [comment]
verify the version of your current Wazuh server using the following 

![image](https://user-images.githubusercontent.com/93060289/164887129-e7010f3e-3fca-4728-97ac-ea9e420f35cc.png)

**Comment by dougburks** [comment, marked as answer]
The Wazuh API is an [Application Programming Interface](https://en.wikipedia.org/wiki/API). It is NOT a user interface.

**Reply by admarnelson** [reply]
Thanks @dougburks  now Understand
